mysql-test/suite/sys_vars/r/sysvars_star.result

```diff
@@ -60,7 +60,7 @@ VARIABLE_NAME	PLUGIN_MATURITY
 SESSION_VALUE	NULL
 GLOBAL_VALUE	alpha
 GLOBAL_VALUE_ORIGIN	CONFIG
-DEFAULT_VALUE	alpha
+DEFAULT_VALUE	experimental
```

Comment:
Rebase on latest 10.7. Serg has fixed this in a follow up commit.

In general, if this is something you need to fix, it needs to go in a separate commit, because it is not related to your push.